compiler/damlc/tests/daml-test-files/Newtype.EXPECTED.md

```diff
@@ -1,26 +1,38 @@
-# Module Newtype
+# <a name="module-newtype-36781"></a>Module Newtype
 
+## Data types
 
+**data <a name="type-newtype-nat-61947"></a>Nat**
 
-## Data types
+* <a name="constr-newtype-nat-99832"></a>Nat
+  
+  | Field | Type/Description |
+  | :---- | :----------------
+  | unNat | Int |
+  
 
-### `data` `Nat`
 
-* `Nat`
+## Functions
 
-  | Field   | Type/Description |
-  | :------ | :----------------
-  | `unNat` | `Int` |
+<a name="function-newtype-mknat-8513"></a>**mkNat**  
+&nbsp; : Int -> [Nat](#type-newtype-nat-61947)
 
+<a name="function-newtype-unsafemknat-96593"></a>**unsafeMkNat**  
+&nbsp; : Int -> [Nat](#type-newtype-nat-61947)
```

Comment:
Should we at least format this in some monospace font?

Reply:
Same as above.